.github/workflows/build-release.yml

```
@@ -0,0 +1,55 @@
+name: CD - Build and release packages
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: GoogleCloudPlatform/release-please-action@v2
+      outputs:
+        released: ${{ steps.release.outputs.release_created }}
+        tag: ${{ steps.release.outputs.tag }}
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        release-type: simple
```

```suggestion
        release-type: simple
        default-branch: main
```
Personellement, je pense qu'on pourrait expliciter la `default-branch`, pour éviter tout problème à l'avenir.

---

> > Si on est vraiment pointilleux, faudra changer "Closes #121" par "Resolves #121, resolves #112"
> 
> Resolves et Closes font la même chose.

En effet, mais je trouve plus approprié "Resolves" pour plusieurs issues

---

Promis je l'ajoute si je dois faire un autre commit ou qu'on me force :upside_down_face:

---

Btw, tu peux juste appuyer sur "Commit suggestion" et ce sera squash au moment du merge